salesagility/SuiteCRM
# Render phone fields as links

## Description
Fixes #7876. This renders phone fields as links.

Before:

<img width="468" alt="Screen Shot 2019-09-17 at 2 46 23 PM" src="https://user-images.githubusercontent.com/2977353/65078293-f22f1780-d959-11e9-8876-2f031678316e.png">

After:
<img width="477" alt="Screen Shot 2019-09-17 at 2 40 49 PM" src="https://user-images.githubusercontent.com/2977353/65077926-3241ca80-d959-11e9-82d4-10c203d67c93.png">

This also includes a minor change to include `phone` classes in phone fields, since the code for doing that existed but wasn't actually functional. I'm honestly not sure why it was done that way, with a special case only for phone fields, but 🤷‍♂ 

## Motivation and Context
Being able to open phone numbers as links is really useful for easily making calls, etc.

## How To Test This
1. Go to a Lead, Contact, Account, etc, make sure they have a phone number associated, or add one yourself.
2. See that the phone number field is now a link.
3. Make sure an empty phone field doesn't have a `tel:` link.
4. Make sure other fields haven't become phone links. 

## Types of changes

### Final checklist
